CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+
+**Important**: Autoprefixer works by pulling data from [caniuse](https://caniuse.com/). Therefore it's important that support for a particular CSS feature is available there before we can enable support for it here. You can check the listing of features that need to be added [here](https://caniuse.com/issue-list).
+
+## Getting started
+Before you begin contributing make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork the repository](https://github.com/postcssAautoprefixer)
+* Clone a copy of it to your computer: `git clone https://github.com/USERNAME/autoprefixer`
+* Ensure that you have the [Yarn](https://yarnpkg.com/) package manager installed
+* run `yarn install` this will install all dependencies needed to run tests
+* File an issue, see [filing issues](#filing-issues)
+* Once issue is discussed and accepted, you can begin making the necessary changes
+
+## Adding a prefix
+We'll explain how would you go about adding a CSS feature to Autoprefixer. For example, we'll add support for a CSS feature called `background-clip: text`
+
+Note: Remember that the feature that you want to add must also be supported on [caniuse](https://caniuse).
+
+* Create a topic branch from the `master` branch. Like this: `git checkout -b add/background-clip-text`
+tip: use 'add', 'fix' or 'enhancement' to indicate on your type of contribution
+
+* To add support for a CSS feature you must call the convert function inside of `data/prefixes.js` and specify three parameters: data, options and a callback. Like this:
+  ```
+  f(require('caniuse-lite/data/features/background-clip-text'), browsers =>
+    prefix(['background-clip'], {
+      feature: 'background-clip-text',
+      browsers
+    })
+  )
+  ```
+* In order to add support for a new feature we need to add a hack to the `libs/hacks` folder using the name of the CSS feature as the filename. Tip: 'hack' is the actual support for the new feature.
+
+* Create a class that extends `Value`. In his new class change the prefix for `IE` to `-webkit-`. [See complete example](https://github.com/postcss/Autoprefixer/blob/73c7b6ab090a9a9a03869b3099096af00be7eb7d/lib/hacks/background-clip.js)
+
+* Load the new class in `lib/prefixes.js`. We refer to a new added class as a 'hack'. To load it we use `Declaration` module and call the `hack` method on the new class we created earlier. Like this:
+  ```
+  Declaration.hack(require('./hacks/background-clip'))
+  ```
+  [See detailed example](https://github.com/postcss/autoprefixer/blob/73c7b6ab090a9a9a03869b3099096af00be7eb7d/lib/prefixes.js)
+
+* Add unit tests to:
+  - `test/autoprefixer.test.js` to test that Autoprefixer works with Chrome and Edge browsers adds only -webkit- prefix. [See details here](https://github.com/postcss/autoprefixer/commit/73c7b6ab090a9a9a03869b3099096af00be7eb7d)
+  - `test/cases/background-clip.css` TODO: Explain how to add unit test to the CSS
```

Comment:
For tests, you need to add `test/cases/PROP.css` with input and `test/cases/PROP.out.css`. If you need specific browsers for the test, add Autoprefixer instance here https://github.com/postcss/autoprefixer/blob/master/test/autoprefixer.test.js#L69. Then add test here https://github.com/postcss/autoprefixer/blob/master/test/autoprefixer.test.js#L424

Reply:
Ok, added this on `0bf7845`